docs/tables/github_code_owner.md

```diff
@@ -0,0 +1,44 @@
+# Table: github_code_owner
+
+All of your code owners rules defined in your repository, in CODEOWNERS files.
+
+The `github_code_owner` table can be used to query information about **ANY** repository, and **you must specify which repository** in the where or join clause (`where repository_full_name=`, `join github_code_owner on repository_full_name=`).
+
+## Examples
+
+### Get All your CodeOwners rules about a specific repository
+
+```sql
+select 
+    row_number() over (order by pattern, users, teams)
+    ,repository_full_name
+    ,pattern
+    ,users
+    ,teams
+    ,pre_comments
+    ,line_comment
+from github.github_code_owner where repository_full_name = 'my-orga/code-owners'
```

Comment:
```suggestion
from github_code_owner where repository_full_name = 'my-orga/code-owners'
```

For examples, we usually don't include the connection name, just the table name is sufficient (you can run queries like this too).

Reply:
Sorry about that, you're right. Fixed